other/materials_designer/create_island_defect.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    def get_coordinate_condition(shape):
</pre><p>Needs to be in made.</p><p> </p><p>Add custom_condition here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>